drivers/misc/ft8xx/ft8xx_copro.c

```diff
@@ -73,6 +78,183 @@ void ft8xx_copro_cmd_swap(void)
 	ft8xx_copro_cmd(CMD_SWAP);
 }
 
+void ft8xx_copro_cmd_fgcolor(uint32_t c)
+{
+	const uint16_t cmd_size = sizeof(CMD_FGCOLOR) +
+				  sizeof(c);
+
+	while (ram_cmd_freespace() < cmd_size) {
```

Comment:
Is this function a blocking call? Probably it should be refelected in documentation as well :) same goes for other functions as well.

Reply:
In an RTOS I expect a function to be blocking unless it's explicitly stated that it's asynchronous and described how the caller is notified when the operation ends.